benchmark_runner/common/prometheus/prometheus_metrics_operations.py

```diff
@@ -160,4 +164,33 @@ def upload_result_to_elastic(self):
                 print(query, results)
         else:
             raise Exception('Missing ElasticSearch data')
-        
+
+    @staticmethod
+    @logger_time_stamp
+    def parse_prometheus_metrics(data: dict):
+        """
+        This method parses prometheus metrics and return summary result
+        based on queries: /benchmark_runner/common/prometheus/metrics-default.yaml
+        @return:
+        """
+        result_dict = {}
+        for query, data_list in data.items():
+            if 'containerCPU-benchmark-runner' in query:
+                suffix = 'CPU'
```

Comment:
This needs to be kept in sync with the metric names.  Since those are under your control, this isn't a problem, just decide whether there might be other similar metrics that you want to do this with.